server/pulp/server/async/celery_instance.py

```diff
@@ -40,13 +40,21 @@
     },
 }
 
-
 celery.conf.update(CELERYBEAT_SCHEDULE=CELERYBEAT_SCHEDULE)
 celery.conf.update(CELERYBEAT_SCHEDULER='pulp.server.async.scheduler.Scheduler')
 celery.conf.update(CELERY_WORKER_DIRECT=True)
 celery.conf.update(CELERY_TASK_SERIALIZER='json')
 celery.conf.update(CELERY_ACCEPT_CONTENT=['json'])
 
+"""
+Configures the celery option CELERYD_MAX_TASKS_PER_CHILD to avoid memory leaks.
+See: https://pulp.plan.io/issues/2172
+"""
+_PULP_MAX_TASKS_PER_CHILD = os.environ.get("PULP_MAX_TASKS_PER_CHILD")
+
+if _PULP_MAX_TASKS_PER_CHILD.isdigit() and _PULP_MAX_TASKS_PER_CHILD >= 1:
+    celery.conf.update(CELERYD_MAX_TASKS_PER_CHILD=_PULP_MAX_TASKS_PER_CHILD)
```

Comment:
Yup, you can either push additional commits, or you can amend this commit and force push it. Either way is fine with me. Also feel free to add yourself to the AUTHORS file!